pyproject.toml

```diff
@@ -8,6 +8,7 @@ description = "A Python interface for the Generic Mapping Tools"
 readme = "README.md"
 requires-python = ">=3.11"
 authors = [{name = "The PyGMT Developers", email = "pygmt.team@gmail.com"}]
+license = {text = "BSD-3-Clause"}
```

Comment:
https://peps.python.org/pep-0639/#deprecate-license-key-table-subkeys

If I understand it correctly, this syntax will be deprecated.

Reply:
Ah ok, I'll revert the change then and mark this PR as draft until setuptools supports PEP639 properly.